### DISCUSSION TITLE
Persistent kill counter

### DISCUSSION THREAD

**Discussion by EXCIDIVM** [diasurgical/DevilutionX#5330] [Ideas]
Hello,
I absolutely LOVE this project. It made me binge on Diablo 1 like a madman again.

Now there is ONE thing I always wanted, ever since I started playing Diablo in 1997 as an angsty teenager: Have a persistent kill counter.
You know, the number of kills it shows you for every creature.

As far as I remember, it always has been persistent in single player. But even in 1997 I didn't bother to play that. In MP, the kill counter gets reset everytime you restart the game.

Would it be possible to add an option for a persistent kill counter? I understand some people might prefer to see the kills per session. But I would absolutely love to see how many thousand skeletons I have slain a few years from now.

**Comment by AJenbo** [comment]
Yeah it's planned, there are already some test builds for it.
